src/include/codegen/plan_comparator.h

```diff
@@ -0,0 +1,73 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// plan_comparator.h
+//
+// Identification: src/include/codegen/plan_comparator.h
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <stdint.h>
+#include <string>
+#include <vector>
+#include "planner/abstract_plan.h"
+#include "planner/abstract_scan_plan.h"
+#include "planner/order_by_plan.h"
+#include "planner/aggregate_plan.h"
+#include "planner/hash_plan.h"
+#include "planner/hash_join_plan.h"
+#include "planner/seq_scan_plan.h"
+
+namespace peloton {
+namespace codegen {
+
+//===----------------------------------------------------------------------===//
+// ExpressionComparator provides compare function for all expressions that are
+// currently supported by codegen.
+//===----------------------------------------------------------------------===//
+class ExpressionComparator {
+
+public:
+  ExpressionComparator() {};
```

Comment:
All functions are **static**, which means no instance is needed. So I wonder if the constructor needs to be disabled (ExpressionComparator() = delete;) in case someone uses it unintentionally.